frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -275,6 +279,10 @@ function FileManagerMenu:setUpdateItemTable()
             end,
             callback = function()
                 G_reader_settings:flipNilOrFalse("dev_no_c_blitter")
+                local InfoMessage = require("ui/widget/infomessage")
+                UIManager:show(InfoMessage:new{
+                    text = _("This will take effect on next restart."),
```

Comment:
Same as above. Something like `The new blitter will take effect after restarting KOReader.` or `KOReader will need to restart to enable the new blitter.`